src/pages/Embeds/Plio.vue

```diff
@@ -659,7 +686,47 @@ export default {
         if (maximizeButton != undefined) this.mountOnFullscreenPlyr(maximizeButton);
       });
     },
+    checkMovingToTimestampAllowed(timestamp) {
+      if (!this.isSkipEnabled && this.lastAnsweredInteractionIndex < this.numItems - 1) {
+        const firstUnansweredInteraction = this.items[
+          this.lastAnsweredInteractionIndex + 1
+        ];
+        if (timestamp >= firstUnansweredInteraction.time)
+          return firstUnansweredInteraction;
+      }
+    },
+    moveToFirstUnansweredItemTimestampOrPass() {
+      if (this.isSkipEnabled) return false;
+      let timeToInspect = this.player.currentTime;
+
+      if (
+        this.isAnyItemActive &&
+        this.player.currentTime < this.currentItem.time &&
+        this.player.currentTime >= this.currentItem.time - POP_UP_CHECKING_FREQUENCY
+      )
+        timeToInspect += POP_UP_CHECKING_FREQUENCY;
+
+      const firstUnansweredInteraction = this.checkMovingToTimestampAllowed(
+        timeToInspect
+      );
+      if (firstUnansweredInteraction != null) {
+        this.setPlayerTime(firstUnansweredInteraction.time - POP_UP_CHECKING_FREQUENCY);
```

Comment:
because our check for whether item should be shown or not checks for a window of POP_UP_CHECKING_FREQUENCY **before** the actual timestamp. Probably doing `-0.1` or something should also work but would be more unintuitive. should I add a comment here?